src/OSPSuite.Core/Domain/Builder/MoleculeStartValue.cs

```diff
@@ -51,7 +51,10 @@ public bool IsPresent
 
       public bool IsEquivalentTo(IMoleculeStartValue moleculeStartValue)
       {
-         var isBaseEquivalent = base.IsEquivalentTo(moleculeStartValue);
+         var isBaseEquivalent = false;
```

Comment:
I don't quite get this code. Were do we use the isEquivalent
and why do we use suddenly an actual instance of PathAndValueEntity. this feels a bit hacky

Reply:
Yeah, the reason for the switch is that I don't want to run this dual inheritance of interfaces and objects. I did not create an IPathAndValueEntity interface at all. But rather than refactor the StartValue classes and tasks to deal with concrete implementations, I drew the line here. All the tasks and whatever are written to use IParameterStartValue.